_includes/header.html

```diff
@@ -1,47 +1,65 @@
 <nav class="navbar" role="navigation" aria-label="main navigation">
   <div class="container">
-    <div class="navbar-brand">
-      <a class="navbar-item" href="{{ '/' | url }}">
-        <img src="{{ site.logoNegative | url }}" alt="{{ title }}">
-      </a>
+    <div class="inner-container">
+      <div class="navbar-brand">
+        <a href="{{ '/' | url }}">
+          <img src="{{ site.logoMark | url }}" alt="{{ title }}">
+        </a>
 
-      <a role="button" class="navbar-burger" data-target="navbar-menu" aria-label="menu" aria-expanded="false">
-        <span aria-hidden="true"></span>
-        <span aria-hidden="true"></span>
-        <span aria-hidden="true"></span>
-      </a>
-    </div>
+        <button id="hamburgerMenu" class="navbar-burger" data-target="navbar-menu" aria-label="menu" aria-expanded="false" type="button">
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+        </button>
+      </div>
 
-    <div class="navbar-menu" id="navbar-menu">
-      {% for link in menu %}
-        {% assign pagedir = page.url %}
-        {% if link.subpages %}
-          {% capture subpage-is-active %}
-            {% for sublink in link.subpages %}
-              {% if sublink.url == pagedir %}
-                is-active
-              {% endif %}
-            {% endfor %}
-          {% endcapture %}
-          <div class="navbar-item has-dropdown is-hoverable">
-            <a class="navbar-link{% if link.url == pagedir %} is-active {% endif %}{{ subpage-is-active }}" href="{{ link.url | url }}"{% if link.url contains 'http://' or link.url contains 'https://' %} target="_blank"{% endif %}>
-              {{ link.title }}
-            </a>
-            <div class="navbar-dropdown">
+      <div class="navbar-menu" id="navbar-menu">
+        {% for link in menu %}
+          {% assign pagedir = page.url %}
+          {% if link.subpages %}
+            {% capture subpage-is-active %}
               {% for sublink in link.subpages %}
-                <a class="navbar-item{% if sublink.url == pagedir %} is-active{% endif %}" href="{{ sublink.url | url }}"{% if sublink.url contains 'http://' or sublink.url contains 'https://' %} target="_blank"{% endif %}>
-                  {{ sublink.title }}
-                </a>
+                {% if sublink.url == pagedir %}
+                  is-active
```

Comment:
Nit: It seems we don't have styles for `is-active`, or they are not being applied (neither in desktop or mobile version of the website).

Reply:
There are some specific styles applied to elements with is-active in the CSS file.

Reply:
But I don't see them taking effect here, not sure what I'm missing.

Reply:
`.navbar-burger.is-active` is used to become the hamburger button on mobile into a X when it's opened.

But for the actual active page (e.g. https://servo.org/servo-redesign/download/) we don't have any kind of effect in the menu (neither on desktop or mobile). Anyway it's a minor thing we can fix in the future.

Reply:
Created an issue to track this: https://github.com/servo/servo.org/issues/208